app/assets/javascripts/item_selector/item_selector_single_select.js

```diff
@@ -0,0 +1,28 @@
+var itemSelectorSingleSelect = (function() {
+
+  return $.extend({}, itemSelector, {
+    init: function() {
+      var _this = this;
+      $(document).on('ready page:load', function(){
+        _this.addDeselectBehaviorForRadioButtons();
+      });
+    },
+
+    addDeselectBehaviorForRadioButtons: function() {
+      var _this = this;
+      var selections = [];
+      _this.selectorElement()
+           .on('item-selector:selected', function(e, checkbox) {
+        if(checkbox.prop('type') == 'radio') {
+          selections.push(checkbox);
+          if(_this.numberOfSelectedCheckboxes() > 1) {
+            _this.selectorElement()
+                 .trigger('item-selector:deselected', [selections[0]]);
```

Comment:
In the case of the radio button there is only ever 1 selected element (which is the currently selected radio when the change event is called). We need to know which was the previously selected element and pass it to the deselected event (since no change event is triggered on the radios that are being deselected).  Since we call `shift()` below, the first element is popped off of the array right after it is passed to the deselected event and the array will only ever contain 2 elements (and only for an extremely brief time before the first is shifted off).